src/components/Rules.js

```diff
@@ -1,13 +1,13 @@
 import React from "react";
 
-import "./rule.css";
+import "./rules.css";
 
 const Rules = props => {
   return (
-    <div className="game-rules__container" onClick={props.onClick}>
-      <div className="game-rules">
-        <h2 className="game-rules__title">Game Rules</h2>
-        <div className="game-rules__content">
+    <div className="game__rules" onClick={props.onClick}>
```

Comment:
So, the block identifier here would be `game-rules`, so the underlying classes would be:

- `game-rules__title`
- `game-rules__content`

... and so on!